test/core-sdk-tasks.Tests/CalculateTemplateVerionsTests.cs

```diff
@@ -11,39 +11,41 @@ public void WhenAspNetCoreTemplateMajorVersionLowerthan3ItCanCalculateTemplateVe
         {
             var result = CalculateTemplateVersions.Calculate("3.1.0");
 
-            result.Should()
-                .Be(("3.1.1", "3.1", "3.1.1"),
-                    "the patch is 1 higher than aspnetTemplateVersion " +
-                    "due to https://github.com/dotnet/core-sdk/issues/6243");
+            // The patch is 1 higher than aspnetTemplateVersion due to https://github.com/dotnet/core-sdk/issues/6243
+            result.InstallPath.Should().Be("3.1.1");
+            result.MajorMinorVersion.Should().Be("3.1");
+            result.MajorMinorPatchVersion.Should().Be("3.1.1");
```

Comment:
These tests used ValueTuple comparison to operate previously. It was easier to compare each property explicitly when using the DTO class instead of creating an equality comparer for 3 properties total.